src/transformers/pipelines.py

```diff
@@ -1738,6 +2003,12 @@ def __call__(
         "pt": AutoModelWithLMHead if is_torch_available() else None,
         "default": {"model": {"pt": "gpt2", "tf": "gpt2"}},
     },
+    "dialogue": {
```

Comment:
can we call this "conversational" to be compatible out of the box with the models that are already tagged on https://huggingface.co/models?filter=conversational ?

I think dialogue might be the better term for the NLP research community but conversational is better for ML practitioners (who are the target audience of pipelines)